test/e2e/config_amd64.go

```diff
@@ -1,15 +1,14 @@
 package integration
 
 var (
-	STORAGE_FS               = "overlay"                                                                                                                   //nolint:revive,stylecheck
-	STORAGE_OPTIONS          = "--storage-driver overlay"                                                                                                  //nolint:revive,stylecheck
-	ROOTLESS_STORAGE_FS      = "overlay"                                                                                                                   //nolint:revive,stylecheck
-	ROOTLESS_STORAGE_OPTIONS = "--storage-driver overlay"                                                                                                  //nolint:revive,stylecheck
```

Comment:
ROOTLESS_STORAGE_OPTIONS was nuked in #18544. (The rest of this block is whitespace-only. Same with the next two config_xxx.go files).